### PR TITLE
FBSnapshotTestCase integration. Initial tests for AttributedStringView

### PR DESCRIPTION
**Pull request by Austinate**
This PR adds (#656): 

- FBSnapshotTestCase integration:
   - Pod for Test scheme
   - Environment variables to main scheme: screen shot 2017-10-23 at 00 33 11
   - Added 'FailureDiffs/' folder signature to .gitignore (we would not want to have images from Failure Diffs in repository)
- 2 simple tests for AttributedStringView (please correct me if there any additional tests needed since i'm not that familiar with codebase yet) :)

### CODE REVIEW

**Comment by BasThomas** [comment]
If we don’t add the snapshots to git, we can only test this locally... is that what we want?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Austinate** [comment]
@BasThomas actual snapshots are in repo. But we won't store failure diffs, since they should be picked up by CI and/or PR author for investigation :)

**Comment by Austinate** [comment]
@BasThomas for example we have [this](https://github.com/Austinate/GitHawk/blob/f058d8c60f03055259df10a572ab6c3687c29e85/FreetimeTests/Snapshot%20Tests/ReferenceImages_64/FreetimeTests.AttributedStringViewTests/testAttributedStringUIEmailIsCorrect%402x.png) in PR (and this should always be part of repo, ofc it doesn't make sense to store reference images locally only),
but if test will fails i will have this in Failure Diffs repo:
screen shot 2017-10-23 at 00 50 28
screen shot 2017-10-23 at 00 51 01

**Comment by BasThomas** [comment]
Ah, obviously. Nevermind me 😔

**Comment by Austinate** [comment]
@BasThomas no worries :)

**Comment by BasThomas** [comment]
Thing is - we use them at work too. Just wasn't thinking. 😅

**Comment by Austinate** [comment]
@rnystrom hi! I see some conflicts already, should i update PR or we're going to merge this later?

**Comment by Austinate** [comment]
Hi, @rnystrom 
Don't want to be annoying, but do we plan to have any actions on this? If so — i'll rebase, if not — probably would be nice to just close.
Thanks in advance!

**Comment by rnystrom** [comment]
Hey! Sorry this fell off my radar. Yes! Let's rebase and I'll land.

**Comment by Austinate** [comment]
@rnystrom Thanks for response! Rebased and ready to go

**Comment by Sherlouk** [comment]
Haven't seen the source for this tweet, but sounds like FBSnapshotTestCase might not be maintained any more?

https://twitter.com/amlcurran/status/933002258790387713

**Comment by Austinate** [comment]
@Sherlouk source is [here](https://github.com/facebookarchive/ios-snapshot-test-case/issues/227#issuecomment-335175385)
tl;dr: `Unfortunately I recently archived the project as we no longer use this internally ... I am working on a solution though to provide a better owner for it. I'll keep you posted.`

So i think there is a big chance that this still will be maintained

**Comment by rnystrom** [comment]
We use this at Instagram, it's a wonderful tool for building UI w/out having to spin up the entire app. Not to mention regression testing.
